### ISSUE TITLE
Inverse Phoneme Decoding

### ISSUE THREAD

**Issue by olivershetler** [fwillett/speechBCI#11]
Hi all! 

My team is trying to understand how you inverted the grapheme to phoneme translation that was implemented using the g2p_en package. We noticed that this package doesn't have an inverse model and we are having trouble finding the details of how transcription from phonemes to text was performed in the codebase and the paper / supplementary. We would be grateful for any pointers to how the phoneme to text transcription was performed.

Cheers,
Oliver Shetler

**Comment by fwillett** [comment]
Hi Oliver,

Thanks for reaching out! The phoneme logits output by the RNN were converted into words using the language model (for example, see this [notebook](https://github.com/fwillett/speechBCI/blob/main/AnalysisExamples/rnn_step3_baselineRNNInference.ipynb)). The language model infers the most likely sequence of words that would explain the phoneme probabilities observed by the RNN. 

Best,
Frank

**Comment by olivershetler** [comment]
Thanks Frank! That clarifies things.
